spec/fixtures/tracks.yml

```diff
@@ -101,3 +101,10 @@ soft_deleted_track_on_asset:
   asset: asset_with_relations_for_soft_delete
   position: 1
   created_at: <%= 2.days.ago %>
+
+track_on_soft_deleted_of_other_user:
```

Comment:
This track fixture isn't yet deleted, which I think makes this name difficult to parse. Maybe something like `track_with_asset_from_other_user` or `sudaras_track_from_arthurs_asset` hmmmm...